addons/project_issue/views/project_project_views.xml

```diff
@@ -24,7 +24,7 @@
         <field name="arch" type="xml">
             <field name="use_tasks" position="after">
                 <field name="use_issues"/>
-                <field name="issue_ids" invisible="1"/>
```

Comment:
Maybe not. I'm not sure if including this field makes this to be less performant, or it's the call to length value the one who actually loads the records.